things/calculator/coap/client-tests/content-negotiation-coap-client.js

```diff
@@ -18,28 +18,27 @@ const fullTDEndpoint = `/${thingName}`,
 /****************************************/
 
 // GET request to retrieve thing description
-function getFullTD() {
+function getFullTD(acceptType) {
```

Comment:
Maybe my personal opinion but I think that the functions should expect the _full_ accept value like
`application/json`

Reply:
I was debating the same while doing it. Thanks for the feedback, I will change it.